scripts/ingests/ingest_BYW_2024.py

```diff
@@ -11,24 +12,31 @@
 
 
 # live google sheet
-link = "https://docs.google.com/spreadsheets/d/1JFa8F4Ngzp3qAW8NOBurkz4bMKo9zXYeF6N1vMtqDZs/pub?output=csv"
+sheet_id = '1JFa8F4Ngzp3qAW8NOBurkz4bMKo9zXYeF6N1vMtqDZs'
 
-columns = ['source', 'ra', 'dec', 'epoch', 'equinox', 'shortname', 'reference', 'other_ref', 'comments']
-byw_table = ascii.read(link, format='ecsv', data_start=2, data_end=90, header_start=1, names=columns, guess=False,
-                           fast_reader=False, delimiter=',')
+df = pd.read_csv(f"https://docs.google.com/spreadsheets/d/{sheet_id}/export?format=csv")
```

Comment:
I said specifically that you don't need the pandas part, just the help with formatting the link.
Use this string as your link: f"https://docs.google.com/spreadsheets/d/{sheet_id}/export?format=csv"
or whatever you learned in the video.  Notice the difference between what you had before:
`pub?output=csv` and what you have now: `pub?output=csv`